cvat-ui/src/components/cloud-storages-page/cloud-storages-page.tsx

```diff
@@ -27,14 +28,7 @@ export default function StoragesPageComponent(): JSX.Element {
     const current = useSelector((state: CombinedState) => state.cloudStorages.current);
     const query = useSelector((state: CombinedState) => state.cloudStorages.gettingQuery);
 
-    const queryParams = new URLSearchParams(history.location.search);
-    const updatedQuery = { ...query };
-    for (const key of Object.keys(updatedQuery)) {
-        (updatedQuery as Indexable)[key] = queryParams.get(key) || null;
-        if (key === 'page') {
-            updatedQuery.page = updatedQuery.page ? +updatedQuery.page : 1;
-        }
-    }
+    const updatedQuery = useUpdatedQuery<CloudStoragesQuery>(query, { pageSize: 12 });
```

Comment:
Maybe `useUpdatedQuery` is not the best name. Just `useResourceQuery<Type>(default)`?
Is there a reason to add `options` argument when `pageSize` and `page` are required in QueryType?

Reply:
Renamed to `useResourceQuery`. Answered about `options` in previous comment.